doc/src/guides/solving.md

```diff
@@ -0,0 +1,45 @@
+# Solve Using the SymPy Python Library
```

Comment:
I know we chose this name for SEO purposes, but this rather verbose title also shows up in the sidebar. I don't know if there's a way to make the one different from the other. I agree it's useful to have this longer name for the title of the page in the Google search results, but once you are already in the SymPy documentation, it actually makes it harder to navigate because of the verbosity.

Reply:
I shortened to _Solve Using SymPy_. The word _Python_ is at the top of the page, so hopefully search engines will take that into account.

Or if people are OK with the grammar, _Solve Using Python's SymPy_ would be a compromise.